src/json/more_related_products0.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
nit: rename this file to more_related_products_page0 (and the others accordingly)

Reply:
done